bin/sdk/gutenberg.js

```diff
@@ -24,12 +26,19 @@ function blockScripts( type, inputDir, presetBlocks ) {
 		.filter( fs.existsSync );
 }
 
-exports.config = ( { argv: { inputDir, outputDir }, getBaseConfig } ) => {
+exports.config = ( { argv: { inputDir, outputDir }, getBaseConfig, calypsoRoot } ) => {
 	const baseConfig = getBaseConfig( {
 		cssFilename: '[name].css',
 		externalizeWordPressPackages: true,
 	} );
 
+	const inputDirInCalypso = inputDir.startsWith( calypsoRoot );
+
+	const commitSha =
+		inputDirInCalypso &&
```

Comment:
Ideally, we would just check if whatever folder we're building from is a git repository and include commit SHA of it, but since it gets a bit complicated I think it's easier to just include it when we know we're inside Calypso directory.